packages/connection/src/Manager.php

```diff
@@ -178,7 +153,7 @@ protected function get_option_manager() {
 	public function generate_secrets( $action, $user_id, $exp ) {
 		$callable = $this->get_secret_callable();
 
-		$secrets = $this->get_option_manager()->get_raw_option( 'jetpack_secrets', array() );
+		$secrets = \Jetpack_Options::get_raw_option( self::SECRETS_OPTION_NAME, array() );
```

Comment:
Nit pick, but can we split the `get_raw_option` argument between multiple lines?

Reply:
```suggestion
		$secrets = \Jetpack_Options::get_raw_option(
			self::SECRETS_OPTION_NAME,
			array()
		);
```